runner/src/internal.rs

```diff
@@ -55,18 +55,15 @@ pub struct Opt {
 
 #[derive(StructOpt, Clone, Debug)]
 pub enum Command {
-    RunExamples(RunExamples),
```

Comment:
This files also has a server variant related to the introspection-client that caused some complication, particularly related to running tests for all features. That, and the code related to it, can be removed too.